saleor/graphql/shop/types.py

```diff
@@ -64,6 +68,19 @@ class ExternalAuthentication(graphene.ObjectType):
     name = graphene.String(description="Name of external authentication plugin.")
 
 
+class Limits(graphene.ObjectType):
+    channels = graphene.Int()
+    orders = graphene.Int()
+    product_variants = graphene.Int()
+    staff_users = graphene.Int()
+    warehouses = graphene.Int()
+
+
+class LimitInfo(graphene.ObjectType):
+    current_usage = graphene.Field(Limits, required=True)
+    allowed_usage = graphene.Field(Limits, required=True)
```

Comment:
Before merge, I would like to add here a description for both, that includes the following:
> [...] `null` has for meaning unlimited.